src/plays/emoji-game/App.css

```diff
@@ -0,0 +1,11 @@
+.app {
```

Comment:
Try to avoid classname collision by adding your play name as a prefix to it, like
`emoji-game-app`